packages/carbon-graphs/tests/unit/controls/Scatter/ScatterLoad-spec.js

```diff
@@ -296,6 +296,100 @@ describe('Scatter - Load', () => {
         input.key,
       );
     });
+    it('does not update x axis range if allow calibration is disabled', () => {
+      const disableCalibrationInput = getAxes(axisDefault);
```

Comment:
I would rename `disableCalibrationInput` to `graphConfig` and
`disableCalibrationGraph` to `graphInstance` like I did here: https://github.com/cerner/terra-graphs/pull/288/files

Reply:
Renamed 25980a5